### ISSUE TITLE
Support Vietnamese character

### ISSUE THREAD

**Issue by haunt98** [0xType/0xProto#44]
Can you add new characters to support Vietnamese
You can get inspire from https://vietnamesetypography.com/

List of chars is here:

```txt
    a á à ả ã ạ ă ắ ằ ẵ ẳ ặ â ấ ầ ẫ ẩ ậ
    A Á À Ả Ã Ạ Ă Ắ Ằ Ẵ Ẳ Ặ Â Ấ Ầ Ẫ Ẩ Ậ
    i í ì ỉ ĩ ị
    I Í Ì Ỉ Ĩ
    u ú ù ũ ủ ụ ư ứ ừ ữ ử ự
    U Ú Ù Ũ Ủ Ụ Ư Ứ Ừ Ữ Ử Ự
    o ó ò õ ỏ ọ ô ố ồ ỗ ổ ộ ơ ớ ờ ỡ ở ợ
    O Ó Ò Õ Ỏ Ọ Ô Ố Ồ Ỗ Ổ Ộ Ơ Ớ Ờ Ỡ Ở Ợ
    y ý ỳ ỷ ỹ ỵ
    Y Ý Ỳ Ỷ Ỹ Ỵ
```

**Comment by masaakim** [comment]
@haunt98 Sure!
I've added support for Turkish and will be extending support to Vietnamese as well. There are more glyphs required for Vietnamese than for Turkish, so I appreciate your patience during this process.

**Comment by masaakim** [comment]
I've opened a new issue, #45, but rest assured, prioritizing support for the Vietnamese language comes first ;)

**Comment by haunt98** [comment]
Thanks @masaakim, take your time! Feel free to close this issue if you want.

**Comment by masaakim** [comment]
I'll roll out the release with the Vietnamese glyphs first, and I'll close this issue once that's done.

**Comment by masaakim** [comment]
Hi @haunt98 

I've just released [v1.403](https://github.com/0xType/0xProto/releases/tag/1.403) to support Vietnamese characters!

**Comment by haunt98** [comment]
Thanks @masaakim, it looks perfect!
